server/src/main/resources/org/cloudfoundry/identity/uaa/db/hsqldb/V4_7_0_3__Add_Key_Provider_Config.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE key_provider_config (
+   id CHAR(36) NOT NULL PRIMARY KEY,
+   identity_zone_id varchar(36) NOT NULL,
+   client_id VARCHAR(256),
+   dcs_tenant_id VARCHAR(256),
+   created TIMESTAMP default current_timestamp NOT NULL
```

Comment:
1. `last_modified` isn't supported today because we don't do an update
2. UAA in general avoids usage of foreign key for performance reasons. The integrity is enforced at the service layer.

Reply:
@bsekar do you know of any data points around the decision on Foreign Keys? For this kind of data I wouldn't expect it to be an issue. Just curious. (@balans)